pkg/visor/visorconfig/values.go

```diff
@@ -180,12 +180,23 @@ func DmsgPtyWhiteList() string {
 func MustPK(pk string) cipher.PubKey {
```

Comment:
I was curious about how `skycoin/skycoin` handles naming the equivalent functions and found that it seems to have us covered already, so no need to reimplement this here.` ./src/cipher/crypto.go:func MustPubKeyFromHex`. At least I would double check we are not reimplementing this here.

Reply:
I don't think so

What is missing is a Set() method for cipher.PubKeys